pkg/pxc/app/pmm.go

```diff
@@ -7,11 +7,33 @@ import (
 )
 
 func PMMClient(spec *api.PMMSpec, secrets string) corev1.Container {
+	ports := []corev1.ContainerPort{{ContainerPort: 7777}}
+
+	for i := 30100; i <= 30200; i++ {
+		ports = append(ports, corev1.ContainerPort{ContainerPort: int32(i)})
+	}
+
 	pmmEnvs := []corev1.EnvVar{
 		{
 			Name:  "PMM_SERVER",
 			Value: spec.ServerHost,
 		},
+		{
+			Name:  "CLIENT_NAME",
+			Value: "pmm-k8s-agent",
+		},
+		{
+			Name:  "CLIENT_PORT_LISTEN",
+			Value: "7777",
+		},
+		{
+			Name:  "CLIENT_PORT_MIN",
+			Value: "30100",
+		},
+		{
+			Name:  "CLIENT_PORT_MAX",
```

Comment:
can we avoid CLIENT_* vars?

Reply:
I was provided with a valid env configuration in https://github.com/Percona-Lab/pmm-submodules/blob/PMM-4417-k8s-button/build/pmm-server-docker/docker-compose.yml#L8-L29

Hope to get rid of all junk in the future, however it shall be kept here for now.